### PR TITLE
landlord-4

### PR DESCRIPTION
**Pull request by chrislovett03**
comfort: 3
completeness :3

### CODE REVIEW

File: controllers/apartments.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # index
 get '/apartments' do
+  @apartment = Apartment.all
```
Comment (jshawl) [inline controllers/apartments.rb:3]: this works! but `@apartments` (plural) would be a better variable name

File: controllers/apartments.rb [hunk 2]
```diff
@@ -8,7 +9,18 @@
   erb :"apartments/new"
 end
 
+post '/apartments/new' do
```
Comment (jshawl) [inline controllers/apartments.rb:12]: should post to `/apartments`. `/apartments/new` is for displaying the form to create an apartment.

File: controllers/apartments.rb [hunk 3]
```diff
@@ -8,7 +9,18 @@
...
+  Apartment.create!(params[:apartment])
+  redirect "/apartments"
```
Comment (jshawl) [inline controllers/apartments.rb:14]: You can also save the variable and redirect to the newly created apartment:

``` rb
@apartment = Apartment.create!(params[:apartment])
redirect "/apartments/#{@apartment.id}"
```

**Comment by jshawl** [comment]
:+1: You're off to a good start! Let me know if you'd like to review this assignment in a one-on-one.

**Comment by chrislovett03** [comment]
Thanks, Jesse! Sounds good.

**Pull request closed without merging**